test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+foreach(test_src test_tester_1.f90
+                 test_tester_2.f90
+                 test_tester_3.f90
+                 test_tester_4.f90
+                 test_tester_5.f90
+                 test_tester_6.f90)
+    get_filename_component(test_name ${test_src} NAME_WE)
```

Comment:
_get_filename_component_ with NAME_WE cuts off the extension, in this case it is always the same extension, but if you have extensions like f, F, f90, F90,... you can't add the extension in the loop like it was done before.